seed/index.js

```diff
@@ -17,18 +17,16 @@ var Nonprofit = app.models.Nonprofit;
 var Job = app.models.Job;
 var counter = 0;
 var challenges = getFilesFor('challenges');
-var offerings = 2 + challenges.length;
 
-var CompletionMonitor = function() {
-  counter++;
-  console.log('call ' + counter);
-
-  if (counter < offerings) {
-    return;
-  } else {
+function completionMonitor() {
+  // Increment counter and exit if all challenges have been checked
+  if (++counter > challenges.length) {
```

Comment:
You shouldn't increment counters with conditionals. Please move this up a line outside the conditional check.